Cymbal-direct-agent-platform-prompt-design-guide.html

```
@@ -0,0 +1,1832 @@
+                    </div>
+                </section>
+
+                <!-- 8. 最終チェックリスト -->
+                <section class="section" id="sec-8">
+                    <div class="section-head">
+                        <span class="section-num">8</span>
+                        <h2>完了前の最終チェックリスト</h2>
+                    </div>
+                    <ul class="checklist">
+                        <li>
+                            <input type="checkbox" disabled />Task 1: プロンプト名が
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)